[2022-07-22 13:00:16,249] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-06-19T00:00:00+00:00 [queued]>
[2022-07-22 13:00:16,278] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-06-19T00:00:00+00:00 [queued]>
[2022-07-22 13:00:16,280] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-22 13:00:16,282] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-07-22 13:00:16,284] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-22 13:00:16,474] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): t2> on 2022-06-19 00:00:00+00:00
[2022-07-22 13:00:16,511] {standard_task_runner.py:52} INFO - Started process 793 to run task
[2022-07-22 13:00:16,558] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'tutorial', 't2', 'scheduled__2022-06-19T00:00:00+00:00', '--job-id', '43', '--raw', '--subdir', 'DAGS_FOLDER/nickdag.py', '--cfg-path', '/tmp/tmpjsfw2dnz', '--error-file', '/tmp/tmp3zwylry9']
[2022-07-22 13:00:16,586] {standard_task_runner.py:80} INFO - Job 43: Subtask t2
[2022-07-22 13:00:16,784] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:528 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2022-07-22 13:00:17,154] {task_command.py:371} INFO - Running <TaskInstance: tutorial.t2 scheduled__2022-06-19T00:00:00+00:00 [running]> on host 2f63307804d5
[2022-07-22 13:00:18,007] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=caroline.kosalka@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=tutorial
AIRFLOW_CTX_TASK_ID=t2
AIRFLOW_CTX_EXECUTION_DATE=2022-06-19T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-19T00:00:00+00:00
[2022-07-22 13:00:18,011] {python.py:173} INFO - Done. Returned value was: goodbye
[2022-07-22 13:00:18,160] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=tutorial, task_id=t2, execution_date=20220619T000000, start_date=20220722T130016, end_date=20220722T130018
[2022-07-22 13:00:18,235] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-22 13:00:18,602] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-07-25 09:18:02,477] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-06-19T00:00:00+00:00 [queued]>
[2022-07-25 09:18:02,513] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-06-19T00:00:00+00:00 [queued]>
[2022-07-25 09:18:02,514] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-25 09:18:02,515] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-07-25 09:18:02,516] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-25 09:18:02,573] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): t2> on 2022-06-19 00:00:00+00:00
[2022-07-25 09:18:02,578] {standard_task_runner.py:52} INFO - Started process 744 to run task
[2022-07-25 09:18:02,601] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'tutorial', 't2', 'scheduled__2022-06-19T00:00:00+00:00', '--job-id', '44', '--raw', '--subdir', 'DAGS_FOLDER/nickdag.py', '--cfg-path', '/tmp/tmpmbchfoku', '--error-file', '/tmp/tmplxnpegq0']
[2022-07-25 09:18:02,610] {standard_task_runner.py:80} INFO - Job 44: Subtask t2
[2022-07-25 09:18:02,657] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:528 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2022-07-25 09:18:02,839] {task_command.py:371} INFO - Running <TaskInstance: tutorial.t2 scheduled__2022-06-19T00:00:00+00:00 [running]> on host 01e6b1df1873
[2022-07-25 09:18:03,323] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=caroline.kosalka@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=tutorial
AIRFLOW_CTX_TASK_ID=t2
AIRFLOW_CTX_EXECUTION_DATE=2022-06-19T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-19T00:00:00+00:00
[2022-07-25 09:18:03,325] {python.py:173} INFO - Done. Returned value was: goodbye
[2022-07-25 09:18:03,519] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=tutorial, task_id=t2, execution_date=20220619T000000, start_date=20220725T091802, end_date=20220725T091803
[2022-07-25 09:18:03,592] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-25 09:18:03,738] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-07-25 12:25:27,359] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-06-19T00:00:00+00:00 [queued]>
[2022-07-25 12:25:27,587] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tutorial.t2 scheduled__2022-06-19T00:00:00+00:00 [queued]>
[2022-07-25 12:25:27,588] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-25 12:25:27,589] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-07-25 12:25:27,590] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-25 12:25:27,667] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): t2> on 2022-06-19 00:00:00+00:00
[2022-07-25 12:25:27,704] {standard_task_runner.py:52} INFO - Started process 643 to run task
[2022-07-25 12:25:27,761] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'tutorial', 't2', 'scheduled__2022-06-19T00:00:00+00:00', '--job-id', '66', '--raw', '--subdir', 'DAGS_FOLDER/nickdag.py', '--cfg-path', '/tmp/tmpczhe59b7', '--error-file', '/tmp/tmp9je46tvn']
[2022-07-25 12:25:27,765] {standard_task_runner.py:80} INFO - Job 66: Subtask t2
[2022-07-25 12:25:27,916] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:528 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2022-07-25 12:25:28,269] {task_command.py:371} INFO - Running <TaskInstance: tutorial.t2 scheduled__2022-06-19T00:00:00+00:00 [running]> on host 929e6a3728d4
[2022-07-25 12:25:28,939] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=caroline.kosalka@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=tutorial
AIRFLOW_CTX_TASK_ID=t2
AIRFLOW_CTX_EXECUTION_DATE=2022-06-19T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-19T00:00:00+00:00
[2022-07-25 12:25:28,942] {python.py:173} INFO - Done. Returned value was: goodbye
[2022-07-25 12:25:29,426] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=tutorial, task_id=t2, execution_date=20220619T000000, start_date=20220725T122527, end_date=20220725T122529
[2022-07-25 12:25:29,539] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-25 12:25:29,780] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
